src/utils/script_args.py

```diff
@@ -44,10 +44,10 @@ def generic_script_init(description: str) -> ScriptArgs:
         default=False,
     )
     parser.add_argument(
-        "--post-cip20",
+        "--pre-cip20",
         type=bool,
-        help="Flag payout should be made according to pre or post CIP 20 "
-        "(temporary during switch over)",
+        help="Flag payout should be made according to pre or post CIP-20. "
+        "Default is set to the current reward scheme",
```

Comment:
Nit, to make it clear what the current reward scheme is: 

```suggestion
        "Default is set to the current reward scheme (CIP-20)",
```